src/app/contact/contact.component.html

```diff
@@ -1,5 +1,25 @@
 <section id="contact">
   <div class="container">
+<<<<<<< HEAD
```

Comment:
It looks like you have some unresolved merge conflicts here that need to back backed out and addressed. This is a great question to ask in the hacktoberfest channel!